PC/winreg.c

```diff
@@ -101,7 +102,10 @@ PyDoc_STRVAR(PyHKEY_doc,
 "\n"
 "Operations:\n"
 "__bool__ - Handles with an open object return true, otherwise false.\n"
-"__int__ - Converting a handle to an integer returns the Win32 handle.");
+"__int__ - Converting a handle to an integer returns the Win32 handle.\n"
+"__enter__, __exit__ - Context manager support for 'with' statement,\n"
+"automatically closes handle.\n"
+"__eq__, __ne__ - Equality comparison based on Windows handle value.");
```

Comment:
```suggestion
"automatically closes handle.");
```

Windows handle value equality is only added on 3.15.